test/frontend/util/get-jira-configuration-helpers.test.ts

```diff
@@ -169,11 +169,6 @@ describe("getInstallation", () => {
 		await RepoSyncState.destroy({ truncate: true });
 	});
 
-	it("Should get installation from repoSyncState", async () => {
-		const result = await getInstallation(GitHubAPI(), subscription);
-		expect(result).toMatchObject(match);
-	});
```

Comment:
this was a duplicate of the one below.  Must have missed it when I removed the feature flag.